Add step for bumping packages when doing RC patch (#3742)
facebook/react-native-website: website/contributing/release-candidate-patch.md

git cherry-pick <commit-hash>
```

### 2. Bump monorepo packages

- Update packages in the monorepo that were modified by the cherry picks by running `yarn bump-all-updated-packages`. All the package bumps should be on the patch level. Read more about the script and how they work [here](./release-updating-packages).

> - CI will take care of releasing the new versions on npm once you push the changes to the `0.XX-stable` branch.

Reviewer: quick note (already mentioned on the side): remember is that in 0.71 it's 2 separate commands, you need to run the bumping one first, then push it up and wait for the packages to get released, then do the alignment one, while in 0.72 it's one command and you just push it after.

(and well then in 0.70 and lower there's nothing 😅)

Author: We won't release an RC for 0.71 and 0.70, so I skipped those steps in this document. However, I added them to the `stable-patch` which we will do for those versions.

> - Update packages in the monorepo that were modified by the cherry picks by running `yarn bump-all-updated-packages`. All the package bumps should be on the patch level. Read more about the script and how they work [here](./release-updating-packages).
> 
> - CI will take care of releasing the new versions on npm once you push the changes to the `0.XX-stable` branch.

Reviewer 2: Following the first point, I'll be more prescriptive here:

```suggestion
Update all packages in the monorepo that were modified by the cherry picks. You can do it by running:
    ```
    yarn bump-all-updated-packages # All the package bumps should be on the patch level
    git push origin 0.XX-stable
    ```
After pushing, the CI will take care to publish the new packages automatically.

**For 0.71: **
// Add instructions here

** For 0.70: **
For version 0.70, packages must be released manually by a Meta engineer. 
```
